### PR TITLE
Mastery Exercise submission

### PR DESCRIPTION
**Pull request by iobg**

### CODE REVIEW

**Comment by iobg** [comment]
@c-stein

File: word-search.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+let {createReadStream}=require('fs');
+let limitTen=require('./limit-ten')
+let hackType=require('./hacker-typer');
+let es =require('event-stream');
+let {inspect}=require('util');
```
Comment (C-Stein) [inline word-search.js]: I can't find where you're using this library

File: word-search.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+let data='';
```
Comment (C-Stein) [inline word-search.js]: I don't think  you  use this variable

**Comment by C-Stein** [comment]
Nice work. Check out my comments on the code, but the only issues I see are including a library and a variable that you never use- which doesn't affect functionality at all.

File: word-search.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+let [,,arg]=process.argv;
+let readStream=createReadStream('/usr/share/dict/words');
+
+readStream
+.pipe(es.split())                  
+    .pipe(es.map(function (data, cb) {
```
Comment (C-Stein) [inline word-search.js]: I wouldn't indent here...

**Comment by iobg** [comment]
@c-stein, oh thanks, I was trying to do something with those and then scrapped them, forgot to remove the declarations

**Comment by C-Stein** [comment]
👍

**Pull request merged**